components/helm-broker/Gopkg.toml

```diff
@@ -36,8 +36,8 @@ required= [
   revision = "bdcc60b419d136a85cdf2e7cbcac34b3f1cd6e57"
 
 [[constraint]]
-  name = "google.golang.org/grpc"
-  branch = "v1.7.x"
```

Comment:
why did you remove the `google.golang.org/grpc`entry?

Reply:
discussed